packages/jaeger-ui/src/components/TracePage/index.test.js

```diff
@@ -699,15 +700,43 @@ describe('<TracePage>', () => {
       expect(track.trackRange.mock.calls).toEqual([[src, range, [0, 1]]]);
     });
   });
+
+  describe('focusSpan', () => {
+    it('calls updateUiFind and focusUiFindMatches', () => {
+      const spanName = 'span1';
+      wrapper.instance().focusSpan(spanName);
+      expect(updateUiFindSpy).toHaveBeenLastCalledWith({
+        history: defaultProps.history,
+        location: defaultProps.location,
+        uiFind: spanName,
+      });
+      expect(focusUiFindMatchesMock).toHaveBeenLastCalledWith(trace, spanName, false);
+    });
+  });
 });
 
 describe('mapDispatchToProps()', () => {
   it('creates the actions correctly', () => {
-    expect(mapDispatchToProps(() => {})).toEqual({
+    expect(mapDispatchToProps(() => {})).toMatchObject({
```

Comment:
There are few more functions because of the usage of:
```
...((bindActionCreators(timelineActions, dispatch) as unknown) as TDispatchPropsTimelineViewer),
```
In the map dispatch see a comment there.